### DISCUSSION TITLE
Switch to disable foreground and background backup during roaming (cellular data)

### DISCUSSION THREAD

**Discussion by PilaScat** [immich-app/immich#12843] [Feature Request]
This weekend I went abroad and even though I had activated the background backup only in wifi when I went to open immich I found everything in upload and the gigabytes went down very quickly, my father did not have the wifi limitation and used much more.
It would be nice to have an extra switch for both background and foreground in roaming

**Comment by dexif** [comment]
It would be better if it would be possible to separately configure the behavior in roaming. And automatically determine whether the phone is in roaming or not.

**Comment by bo0tzz** [comment]
#20302
